crates/re_viewport/src/viewport_blueprint_ui.rs

```diff
@@ -81,6 +84,137 @@ impl Viewport<'_, '_> {
             });
     }
 
+    /// Expend all required items and compute which item we should scroll to.
+    fn handle_focused_item(&self, ctx: &ViewerContext<'_>, ui: &egui::Ui) -> Option<Item> {
+        ctx.focused_item.as_ref().and_then(|focused_item| {
```

Comment:
prevent rightward drift with this weird trick

```suggestion
        let focused_item = ctx.focused_item.as_ref()?;
```

Reply:
good one 👍🏻